src/lib.rs

```diff
@@ -572,7 +574,15 @@ impl<C: Context> Secp256k1<C> {
 
     /// Returns the required memory for a preallocated context buffer in a generic manner(sign/verify/all)
```

Comment:
please add comment that this returns the size in AlignType units

Reply:
Good Point!

Reply:
I think maybe they should return bytes.

Reply:
why?